README.md

```diff
@@ -6,18 +6,39 @@ Package manager that can be used to install (ROS) dependencies
 
 ## Installation
 
-### Ubuntu 20.04
+### Ubuntu 20.04/22.04
 
 Standard tue-env installation with targets from [tue-env-targets](https://github.com/tue-robotics/tue-env-targets)
 
+#### Add SSH key to GitHub to gain access to this repository
+Add the public part of your ssh-key (`cat ~/.ssh/<KEY_NAME>.pub`, where `<KEY_NAME>` is the name of your ssh-key) to GitHub > Settings > SSH and GPG keys and `New SSH key`
+
+To generate a new ssh keypair:
 ```bash
-source <(wget -O - https://raw.githubusercontent.com/tue-robotics/tue-env/master/installer/bootstrap.bash)
-tue-get install tue-dev #or
-tue-get install tue-dev-full #tue-dev plus extra tools
-tue-make
-source ~/.bashrc
+sudo apt-get install ssh
+ssh-keygen -o -a 100 -t ed25519
+cat ~/.ssh/<KEY_NAME>.pub
```

Comment:
We use RSA by default.

Reply:
This change does not influence what we use as default. Besides, our target only generates a RSA key if does not exist, it does not really make us use it :smile:

Reply:
But people tend to follow all steps, also when they don't need to....

Also it should probably be moved down.